src/main/java/frc/robot/vision/TurnToTarget.java

```diff
@@ -30,7 +30,7 @@
   public TurnToTarget(Target target, MoveableSubsystem subsystem) {
     addRequirements(subsystem);
     this.target = target;
-    output = subsystem::moveWithVision;
+    output = subsystem::move;
```

Comment:
inline the variable output - this means instead of calling `output.accept(pidOutput)`, call `subsystem.move(pidOutput)`.
There isn't any reason to use a consumer here

Reply:
Also, add at the end function `subsystem.stopMove()` instead of `output.accept(0.0)`